emails/2024-05-31.md

```diff
@@ -0,0 +1,25 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+After Zach's [controversial blog about calling variables ok](https://www.dolthub.com/blog/2024-05-10-ok-considered-harmful/), a reader sent me the following joke which I enjoyed, "I thought the three hard problems were exactly-once message delivery, naming things, and exactly-once message delivery...". Keep the jokes coming. It's the hardest part of writing this email.
+
+### Big Numbers
+
+[Neil](https://www.dolthub.com/team#james) wrote [a meme-filled blog about big numbers](https://www.dolthub.com/blog/2024-05-28-160-bits/). It did [pretty well on Reddit](https://www.reddit.com/r/programming/comments/1d2nxvb/hash_collisions_how_large_is_a_160_bit_number/) because it's the type of post that does. 
```

Comment:
```suggestion
[Neil](https://www.dolthub.com/team#neil) wrote [a meme-filled blog about big numbers](https://www.dolthub.com/blog/2024-05-28-160-bits/). It did [pretty well on Reddit](https://www.reddit.com/r/programming/comments/1d2nxvb/hash_collisions_how_large_is_a_160_bit_number/) because it's the type of post that does. 
```